### PR TITLE
Fix various spelling issues in the documentation

### PR DESCRIPTION
**Pull request by Pixep**
Addresses obvious typos, in the documentation only.

### CODE REVIEW

File: pkg/amutex/amutex_test.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func TestMutualExclusion(t *testing.T) {
 	// goroutines ran concurrently within the critical section.
 	//
 	// If one of the goroutines doesn't complete, it's likely a bug that
-	// causes to to wait forever.
+	// causes to wait forever.
```
Comment (prattmic) [inline pkg/amutex/amutex_test.go]: I think this should be "caused it to wait forever".

File: pkg/urpc/urpc.go [hunk 2]
```diff
@@ -72,7 +72,7 @@ type FilePayload struct {
 	Files []*os.File `json:"-"`
 }
 
-// ReleaseFD releases the indexth FD.
```
Comment (prattmic) [inline pkg/urpc/urpc.go]: nit: I think this was correct, though this isn't a real word. Perhaps it should be reworded "ReleaseFD releases the FD at the specified index."

File: pkg/amutex/amutex_test.go (hunk 1 above)
Comment (Pixep) [reply pkg/amutex/amutex_test.go]: Done.

File: pkg/urpc/urpc.go (hunk 2 above)
Comment (Pixep) [reply pkg/urpc/urpc.go]: Ho right! I did not get it the first time. Done.

**Comment by Pixep** [comment]
@prattmic Let me know if you want me to rebase it to fix the conflicts. That's inherent to the number of files affected.

**Comment by Pixep** [comment]
Rebased to fix merge conflicts.

**Comment by prattmic** [comment]
No need, I'm importing this change to merge and already resolved conflicts.

**Comment by prattmic** [comment]
This was merged in https://github.com/google/gvisor/commit/5b41ba5d0eca266790dba5f5dd095010e3944726.

Unfortunately our tooling is slightly broken at the moment, so I ended up as the author and not you. Sorry about that.

**Pull request closed without merging**